scene/gui/item_list.h

```diff
@@ -131,6 +131,7 @@ class ItemList : public Control {
 
 public:
 	void add_item(const String &p_item, const Ref<Texture2D> &p_texture = Ref<Texture2D>(), bool p_selectable = true);
+	void re_add_item(const String &text, const unsigned int &position, const bool &is_enabled, const Ref<Texture2D> &texture, bool p_selectable = true);
```

Comment:
Parameters/arguments should have the `p_` prefix, as you see in other methods. The "Text" here is named `text`, but in other places it is named `name`. I think these should be unified.